packages/memory/integration-tests-v5/src/input-processors.test.ts

```
@@ -0,0 +1,555 @@
+
+    // Second message
+    await agent.generate('I live in Paris', {
+      threadId,
+      resourceId,
+    });
+
+    // Verify messages were saved
+    const { messages: savedMessages } = await memory.recall({ threadId });
+
+    if (savedMessages.length < 4) {
+      debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)